include/rocksdb/db.h

```diff
@@ -905,6 +905,9 @@ class DB {
   virtual Status Flush(const FlushOptions& options) {
     return Flush(options, DefaultColumnFamily());
   }
+  virtual Status Flush(
```

Comment:
may be worth documenting the contract - each of `column_families` will be flushed up through some sequence number greater than or equal to the DB seqnum at the time the call begins.

Reply:
Sure, will do